### PR TITLE
File Download Backend API

### PR DESCRIPTION
**Pull request by UmamaSumlinTasnuva**
- added both file download controller and file download service files
- feature works properly return proper HTTP status code and output in JSON after querying DB

### CODE REVIEW

File: Data Polish-Backend/Data-Polish-Backend/src/main/java/com/example/Data/PolishBackend/Service/FileDownloadService.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    public ResponseEntity<String> getFileDetails(String jobID) {
+        String sql = "SELECT rawurl, dataprofileoutput, datacleaningoutput FROM jobsandblobs WHERE jobid = ?";
+        try {
+            // Execute the query to retrieve the file details
+            String fileDetails = jdbcTemplate.queryForObject(sql, (rs, rowNum) -> {
+                String rawurl = rs.getString("rawurl");
+                String dataprofileoutput = rs.getString("dataprofileoutput");
+                String datacleaningoutput = rs.getString("datacleaningoutput");
+                return "{\"rawurl\":\"" + rawurl + "\", \"dataprofileoutput\":\"" + dataprofileoutput + "\", \"datacleaningoutput\":\"" + datacleaningoutput + "\"}";
```
Comment (D22124491-NaveenMaheswaran) [inline Data Polish-Backend/Data-Polish-Backend/src/main/java/com/example/Data/PolishBackend/Service/FileDownloadService.java:22]: can you use string builder in here

**Review by D22124491-NaveenMaheswaran — APPROVED** [review]
Looks good, Just make this change in next sprit as part of stabilization

**Review by SeanMcCrossanTUD — APPROVED** [review]
LG would let @D22124491-NaveenMaheswaran  comment on the quality of code

File: Data Polish-Backend/Data-Polish-Backend/src/main/java/com/example/Data/PolishBackend/Service/FileDownloadService.java (hunk 1 above)
Comment (UmamaSumlinTasnuva) [reply Data Polish-Backend/Data-Polish-Backend/src/main/java/com/example/Data/PolishBackend/Service/FileDownloadService.java:22]: will do that in later version when we enhance the quality of code

**Pull request merged**